src/components/ProfileValidation/VerifyIdentify.tsx

```diff
@@ -14,27 +18,30 @@ export type VerifyIdentifyProps = {
 export const VerifyIdentify = (props: VerifyIdentifyProps) => {
   const { verificationSubmission } = props
   const [attachments] = useState(verificationSubmission.attachments)
-  const [showORCiDDialog, setShowORCiDDialog] = useState(false)
+
   useEffect(() => {
     verificationSubmission.attachments = attachments
   }, [attachments, verificationSubmission])
 
   return (
     <>
-      <Box sx={{ marginBottom: '48px' }}>
-        {verificationSubmission.orcid && (
+      <Box sx={{ marginBottom: theme.spacing(5) }}>
+        {verificationSubmission.orcid ? (
           <>
-            <Typography style={{ display: 'flex' }} variant="body2">
+            <Typography
+              style={{ display: 'flex', marginTop: theme.spacing(5) }}
+              variant="body2"
+            >
               <CheckIcon style={{ color: '#32A330', marginRight: '8px' }} />
               Profile linked successfully {/*verificationSubmission.orcid*/}
             </Typography>
+            {/* per Adam - they can unlink it from the account page, so we don't need this button here 
```

Comment:
moving 'unlink' from here. Adam mentioned moving it to the profile/account page

Reply:
Looks like Unlink on the account page is currently unavailable (I only see a Link ORC Id button for my test user who does have a linked ORC Id account).